### DISCUSSION TITLE
I would like to simulation by reading the initial configuration of particles and rigid bodies.

### DISCUSSION THREAD

**Discussion by fluidkocy** [InteractiveComputerGraphics/SPlisHSPlasH#263] [Q&A] [answered]
First of all, thank you for creating such a great simulation software and providing it for free.

　I understand that by pressing "ctrl s" I can record the state at a certain time, by pressing "ctrl l" I can record the state at that point, and by pressing "ctrl l" I can read the continuation. But it is only a continuation from that point.

　 I want to create a scene file with the state of "ctrl s" as t = 0. In other words, I want to read the ongoing particle and rigid body states and simulate them from t = 0.

　I would appreciate any and all advice, any suggestions would be appreciated. Please. If you have any questions about my question, please feel free to ask.

**Comment by janbender** [comment, marked as answer]
The state stores the positions and velocities of all particles in the bgeo file. You could also load this file in a scene as initial state. Something like this: 
```
"FluidModels": [
		{
			"particleFile": "state.bgeo",
			"translation": [-2.0, 0.1, 0.0],
			"rotationAxis": [0, 1, 0],
			"rotationAngle": 0,
			"scale": [1, 1, 1]
		}
	]
```

However, the state does not store the dynamic rigid body state since this is part of an external lib.

**Reply by fluidkocy** [reply]
Thank you very much.  Your answer suits me well !!
